### PR TITLE
Burrough 5-7

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: life/modernized/06-chapter-5.adoc [hunk 1]
```diff
@@ -181,18 +192,30 @@ that his dread may go along with us, over all.
 As you can, at opportunity, write to us, it will make me glad;
 the face of one Friend would rejoice my soul.
 
+--
+
+[.offset]
 In a postscript he adds,
```
Comment (maijapapaya) [inline life/modernized/06-chapter-5.adoc:198]: I had at first use the [.postscript] thing for this, but I thought it looked better this way. But maybe you want it as a postscript, just let me know.

File: life/modernized/06-chapter-5.adoc [hunk 2]
```diff
@@ -181,18 +192,30 @@ that his dread may go along with us, over all.
...
 
+[.embedded-content-document.letter]
```
Comment (maijapapaya) [inline life/modernized/06-chapter-5.adoc:200]: I had at first used the [.postscript] thing for this, but I thought it looked better this way. But maybe you want it as a postscript, just let me know.

**Comment by maijapapaya** [comment]
I wasn't entirely sure in chapters 6 and 7 of whether some things should be labeled as letters or addresses, so a few might have to be switched.

**Comment by Henderjay** [comment]
Looks good. Merged 👍🏻

**Pull request merged**